src/standard.js

```diff
@@ -47,6 +47,19 @@ export const onSummonHandBuff = {
     }
   ]
 };
+
+export const onSummonBounce = {
+  ...standard(1),
+  emoji: "",
+  name: "Bouncer",
+  text: "onSummon: choose a friendly creature to return to your hand",
+  onSummon: [
+    {
+      type: "BOUNCE",
+      target: { type: "creature", count: 1, location: "field", player: "self" }
```

Comment:
looks like this player targetting doesn't work right now but that's not the fault of this PR and idc